ultralytics/utils/callbacks/wb.py

```diff
@@ -127,11 +130,27 @@ def _log_plots(plots, step):
 
 def on_pretrain_routine_start(trainer):
     """Initialize and start wandb project if module is present."""
+    if trainer.args.project is None:
+        project_name = "Ultralytics"
+    else:
+        project_str = str(trainer.args.project)
+        project_arg = Path(project_str)
+        project_name = project_arg.name if project_arg.is_absolute() else project_str
+        project_name = project_name.replace("/", "-")
+
+    raw_name = str(trainer.args.name).replace("/", "-") if trainer.args.name else None
```

Comment:
trainer.args.name cannot be none also ultralytics train already set to "trainX" by default and increase the number so we don't have to check this "none" part so you can remove "if" entirely